Xamarin.Forms.Platform.WPF/Properties/AssemblyInfo.cs

```diff
@@ -30,7 +29,9 @@
 [assembly: ExportRenderer(typeof(ActivityIndicator), typeof(ActivityIndicatorRenderer))]
 [assembly: ExportRenderer(typeof(Frame), typeof(FrameRenderer))]
 [assembly: ExportRenderer(typeof(ListView), typeof(ListViewRenderer))]
+#if !NETCOREAPP3_0
 [assembly: ExportRenderer(typeof(OpenGLView), typeof(OpenGLViewRenderer))]
```

Comment:
Technically, the opengl will work. I get that annoying warning, but it still works just fine for SkiaSharp.

Reply:
Hi Matthew, 

why do you think it should work? The OpenGlViewRenderer uses OpenTK which does not support .net core, thats why I left it out.

Reply:
you can run application only on Windows, where net framework is installed and open gl is compatible with it

Reply:
Yeah OpenGL is but OpenTK is not compatible with .net core 3.1 and will therefore not compile :)

Reply:
Well, technically it does. OpenTK will install into netcoreapp because net45+ also is netcoreapp3.0. You will get a NuGet warning saying that the package was "accepted" but the library author needs to multi-target.

Reply:
While I can add the NuGet to the project, I cant build it anymore. The errors goes like `error NU1701 package opektk was not resolved for the target farmework netcoreapp 3.1 but netframework 4.6.1`. I'm a bit confused how I could resolve that issue.

Reply:
Translated error message: `error NU1701: The package "OpenTK 3.1.0" was not created with the project target framework ".NETCoreApp,Version=v3.1", but with ".NETFramework,Version=v4.6.1, . NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8" was restored. This package may not be fully compatible with your project.`

Reply:
Ah, I think `NU1701` is a warning, but maybe the project has warnings as errors? 

I added `NoWarn="NU1701"` to the PackageReference tag. Not sure what the rule are for this in Xamarin.Forms.

SkiaSharp does it like this: https://github.com/mono/SkiaSharp/blob/master/source/SkiaSharp.Views/SkiaSharp.Views.WindowsForms/SkiaSharp.Views.WindowsForms.csproj#L11-L12

Reply:
Yea, we have treat warnings as errors enabled but do ignore warnings in a few places. So, if it's best to just ignore this warning that's fine

Reply:
I added the warn ignore and included the dependency and view for .netcoreapp3.1 again.